desktop/extensions.md

```diff
@@ -103,3 +103,52 @@ If an extension does not provide a way for you to give feedback, contact us and
 3. Click **Uninstall**.
+
+## Install an extension which is not in the Marketplace
+
+The Extensions Marketplace is the trusted and official place to install extensions from within Docker Desktop. These extensions have gone through a review process and thus meet a minimum set of requirements. However, the Marketplace it is not the only place where to install extensions from.
+
+Given the nature of a Docker Extension (i.e. a Docker image) you could find other places where users have their extensions' source code published: GitHub, GitLab or even hosted in image registries like DockerHub or GHCR.
+You can install Docker Extensions that have been developed by the community or internally at your company from a teammate: you are not limited to install extensions from the Marketplace only.
+
+> **Note**
+>
+> Ensure the option "Allow only extensions distributed through the Docker Marketplace" is disabled. Otherwise this will This will prevent to install any other extension via the Extension SDK tools.
+> You can change this option in the Settings > Extensions.
+
+
+> **Warning**
+>
+> Docker Extensions that are not in the Marketplace haven't been gone through a review process by Docker.
+> Extensions can install binaries, invoke commands and access files on your machine. Installing them is at your own risk.
+
+To install an extension which is not present in the Marketplace, you can use the Extensions CLI that is bundled as part of Docker Desktop.
+
+In a terminal, type `docker extension install IMAGE[:TAG]` to install an extension by its image reference and optionally a tag. Use the `-f` or `--force` flag to not prompt for confirmation.
+
+Go to the Docker Dashboard to see the new extension installed.
+
+## List installed extensions
+
+Regardless whether the extension was installed from the Marketplace or manually by using the Extensions CLI, you can use the `docker extension ls` command to display the list of extensions installed.
+As part of the output you could see the extension ID, the provider, version, the title and whether it runs a backend container or has deployed binaries to the host, for example:
+
+```
+$ docker extension ls
+ID                                       PROVIDER            VERSION             UI                              VM                  HOST
+docker/volumes-backup-extension          Docker Inc.         1.0.0               1 tab(Volumes Backup & Share)   Running(1)          1 binarie(s)
+```
+
+Go to the Docker Dashboard, click on "Add Extensions" and on the "Installed" tab to see the new extension installed.
+Notice that it will display an `UNPUBLISHED` label indicating that the extension has not been installed from the Marketplace.
+
+## Update an extension which is not in the Marketplace
+
+To update an extension which is not present in the Marketplace, in a terminal type `docker extension update IMAGE[:TAG]` where the `TAG` should be different from the extension that is already installed.
+
```

Comment:
Notice that after installing extensions that are not published in the Marketplace, you will not get update notifications from Docker Desktop

Reply:
Added a note about this